package/environment.js

```diff
@@ -76,6 +79,14 @@ const getBaseConfig = () =>
 
     resolveLoader: {
       modules: ['node_modules']
+    },
+
+    node: {
+      dgram: 'empty',
+      fs: 'empty',
+      net: 'empty',
+      tls: 'empty',
+      child_process: 'empty'
```

Comment:
What is this for?

Reply:
So, some libraries  that import nodejs (like `fs`) modules but don't use them in the browser. Here, we are telling webpack to provide empty mocks for them so importing works.

Reply:
https://webpack.js.org/configuration/node/#other-node-core-libraries

Reply:
@gauravtiwari Is this still needed? Neither [the docs](https://webpack.js.org/configuration/node/#other-node-core-libraries) or [NodeStuffPlugin](https://github.com/webpack/webpack/blob/master/lib/NodeStuffPlugin.js) seem to use it!